Cargo.toml

```diff
@@ -106,6 +106,7 @@ base64 = "0.10"
 safemem = "0.3"
 cop_datatype = { path = "components/cop_datatype" }
 smallvec = { version = "0.6", features = ["union"] }
+procinfo = { git = "https://github.com/tikv/procinfo-rs" }
```

Comment:
I see that we use this because of https://github.com/danburkert/procinfo-rs/pull/42 (which is failing on CI) and https://github.com/tikv/procinfo-rs/pull/5 (no master PR).

Is it the plan to update this to master later?

Reply:
> no master PR

Yes there is, danburkert/procinfo-rs#38

> which is failing on CI

It's because we're using a very recent Rust version, while procinfo supports Rust version down to 1.13, which is an ancient version.